src/components/UI/FilterMenu/CommonFilterMenu.tsx

```diff
@@ -37,9 +40,18 @@ function CommonFilterMenu<ValueType>({
   const handleMenuItemClick = (event: MouseEvent<HTMLLIElement>) => {
     const value = (event.currentTarget.dataset.value ?? null) as ValueType;
     setSelectedValue(value);
+    addParameterToUrl(
+      parameterName,
+      (event.currentTarget.dataset.value ?? null) as string
```

Comment:
i think it is not meaningful to set a null as string

Reply:
fixed on commit a3cb348c2e28945e4ace5269648f8523807b427b